draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -795,6 +795,16 @@ This document has no actions for IANA.
 </section>
 </section>
 
+<section title="Prefix-Specific Link-Local Addresses">
+<t>
+Various router vendors already allow administrators to configure multiple link-local addresses on a given interface (e.g. configuring an interface link-local and multiple VRRP link-local addresses).
+Some implementations are known to use multiple addresses to send RAs (but currently each RA includes all PIOs, instead of only one).
+</t>
+<t>
+Statically configuring prefix-specific link-local addresses has been successfuly deployed in large-scale IPv6-mostly network to mitigate the impact of flash renumebring and host mobility.
```

Comment:
What are the  host implementations that can readily benefit from this behavior?  It seems that , somehow, this behaviour makes flash-renumbering a multi-homing problem.... but then multi-{homing,prefix,router} is usually considered as broken.... hence the question...

Reply:
I've just submitted an update providing details. More details can be found in my "Mission Possible" talks (see https://ripe87.ripe.net/wp-content/uploads/presentations/32-IPv6-Mostly-Office_-JenLinkova_RIPE87.pdf starting from "Discovery 5.1" slide)

This worked like a charm - users stopped complaining. That's the whole reason I'm suggesting this - I did it and it worked.

Reply:
I've checked the slides. But at least according to the specs, Rule 5.5 doesnt seem to imply that because the next hop is unreachable, the src address should not be used.... 

Also: do Android and Mac implement RFC8028? -- My understanding is that RFC8028 wasn't implemented....

(Is it actually that, as a result of RFC6059, the old address and router are actually removed?)

Reply:
> I've checked the slides. But at least according to the specs, Rule 5.5 doesnt seem to imply that because the next hop is unreachable, the src address should not be used....

as per 6.3.6 of RFC4861 yes, it how it works.

> Also: do Android and Mac implement RFC8028? -- My understanding is that RFC8028 wasn't implemented....

8028 is not needed here. Apple implements Rule 5.5

> (Is it actually that, as a result of RFC6059, the old address and router are actually removed?)

For Android - yes, it detects a new link-local address for the gateway and just reconnects.
Apple devices stay connected and keep both old and new prefixes, both preferrred, but the "new" one is used.
(well, there is a bug when they do not update the clat address, but it's a different story)